### PR TITLE
Create gel-foundations component and delete gel-constants and gel-foundations-styled-components

### LINKED ISSUE #178: Export Strings instead of styled-components CSS
**Is your feature request related to a problem? Please describe.**
Following #142, it is clear we can export String values from packages such as `gel-foundations-styled-components`, instead of styled-components CSS components. This removes an unnecessary dependency from us, & from anybody who wishes to utilise these foundational packages.

**Describe the solution you'd like**
Instead of:
```
export const GEL_MINION = css`
  font-size: 0.75em;
  line-height: 1em;
`;
```
We would do:
```
export const GEL_MINION = `
  font-size: 0.75em;
  line-height: 1em;
`;
```

[Because of our current usage of these values](https://github.com/BBC-News/psammead/blob/a33a2e3afd9debc464b39c8d5b16624920a89945/packages/components/psammead-paragraph/src/index.jsx#L7-L13), which already injects the exported values into styled-components CSS blocks where used, there should be no change to other packages as a result of this change alone.

However, it raises an interesting point as to our current packages. The `gel-` packages are currently split primarily by implementation detail -- whether `-styled-components` or `-constants`. Following this PR, there will be no need for the `-styled-components` package any more. As such, one of the following should happen:
1. All type size exports to move to `gel-constants`
2. Both `gel-foundations-styled-components` & `gel-constants` become a new `gel-foundations` package.

I would recommend the second option, as this simplifies the "product-y" view of our utility packages.

**Testing notes**
[Tester to complete]

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #178 

Create gel-foundations component and delete gel-constants and gel-foundations-styled-components

When this is approved I will add a message to the top of `gel-constants` and `gel-foundations-styled-component` Readmes explaining to use the new package, publish them and deprecate the packages in npm

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/utilities/gel-foundations/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Gel Foundations Changelog
+
+| Version | Description |
+|---------|-------------|
+| 0.1.0   | [PR#XX](https://github.com/BBC-News/psammead/pull/XX) Create initial package, pulled in from gel-constants and gel-foundations-styled-components packages. |
```
Comment (ChrisBAshton) [inline packages/utilities/gel-foundations/CHANGELOG.md]:
```suggestion
| 0.1.0   | [PR#221](https://github.com/BBC-News/psammead/pull/221) Create initial package, pulled in from gel-constants and gel-foundations-styled-components packages. |

```

File: packages/utilities/gel-foundations/README.md [hunk 2]
```diff
@@ -1,18 +1,22 @@
-# gel-foundations-styled-components &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/gel-foundations-styled-components.svg)](https://www.npmjs.com/package/@bbc/gel-foundations-styled-components) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
+# gel-foundations &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/gel-foundations.svg)](https://www.npmjs.com/package/@bbc/gel-foundations) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
 
-This package is a collection of styled component implementations based on BBC GEL guidelines which can be imported into your application.
+This package is a collection of variable constants which can be imported into your application.
```
Comment (ChrisBAshton) [inline packages/utilities/gel-foundations/README.md]:
```suggestion
This package is a collection of constants which can be imported into your application.
```

"variable constants" sounds like an oxymoron :-)

**Review by ChrisBAshton — CHANGES_REQUESTED** [review]
Looks good! Just a tweak or two

**Review by ChrisBAshton — APPROVED** [review]

**Review by bcmn — APPROVED** [review]
Approving -- This is going to conflict a whole bunch with my work on #212, so I'll block that ticket on this one.

**Pull request merged**